drivers/i2c/i2c_mcux.c

```diff
@@ -134,12 +135,13 @@ static int i2c_mcux_transfer(struct device *dev, struct i2c_msg *msgs,
 
 		/* Start the transfer */
 		status = I2C_MasterTransferNonBlocking(base,
-				&data->handle, &transfer);
+						       &data->handle, &transfer);
 
 		/* Return an error if the transfer didn't start successfully
 		 * e.g., if the bus was busy
 		 */
 		if (status != kStatus_Success) {
+			I2C_MasterTransferAbort(base, &data->handle);
```

Comment:
As far as I can tell, there should only be to lines added for this PR, namely this one...